client/src/partials/profit_center/center/analysis_profit_center.js

```diff
@@ -9,7 +9,8 @@ angular.module('bhima.controllers')
   'SessionService',
   function ($scope, connect, appstate, messenger, validate, $translate, SessionService) {
```

Comment:
Remove unused services such as ``` appstate ```.
I think after aliasing your controller, $scope will not be usefull